docs/launch_modes.md

```diff
@@ -20,28 +20,27 @@ modes along with Triton Server. In the `local` and `docker` modes, Triton
 Inference Server will be launched by the Model Analyzer. In the `remote` mode,
 it is assumed there is an already running instance of Triton Inference Server.
 
-1. **`tritonserver` binary is available in `$PATH`**. In this case you can use
-   the `--triton-launch-mode local` flag. Model Analyzer will launch a Triton
-   Inference Server that will be used for benchmarking the models.
-
-2. **Using Docker API to launch Triton Inference Server container.** If you are
-   using this mode and you are using Model Analzyer inside a Docker container,
-   make sure that the container is launched with appropriate flags. The
-   following flags are mandatory for correct behavior:
+1. **`--triton-launch-mode=local`** In this mode, Model Analyzer will launch
+   Triton Server using the local binary supplied using `--triton-server-path`,
+   or if none is supplied, the `tritonserver` binary in `$PATH`.
+
+2. **`--triton-launch-mode=docker`** In this mode, Model Analyzer uses the
+   python Docker API to launch the Triton Inference Server container. If you are
+   running Model Analyzer inside a Docker container, make sure that the
+   container is launched with appropriate flags. The following flags are
+   mandatory for correct behavior:
```

Comment:
Could you also add a note here about the "output_model_repository" considerations for Docker mode?